start/resources/requirements.md

```diff
@@ -74,44 +71,37 @@ Below is a list of common attributes that you can include in your submit file `r
 - **OSGVO_OS_VERSION** - Version of the operating system
 
 - **OSGVO_OS_STRING** - Combined OS name and version. Common values are
-  _RHEL 6_ and _RHEL 7_. Please see the requirements string above on the
+  _RHEL 7_ and _RHEL 8_. Please see the requirements string above on the
   recommended setup.
 
 - **OSGVO_CPU_MODEL** - The CPU model identifier string as presented in
   /proc/cpuinfo
-- **CVMFS_oasis_opensciencegrid_org_REVISION** - Attribute specifying
-  the need to access specific oasis /cvmfs file system repositories. See the
-  example above for determining the appropriate REVISION number when using
-  this `requirements` attribute.
+
+- **HAS_CVMFS_oasis_opensciencegrid_org** - Attribute specifying
+  the need to access specific oasis /cvmfs file system repositories.
 
 - **CUDACapability** - For GPU jobs, specifies the CUDA compute capability.
+  See our [GPU guide](5000653025) for more details.
+
+# Specifying Sites / Avoiding Sites
 
-- **HAS_SQUID** - Boolean specifying the need for access to SQUID fileshare,
-  set to `true` if your job has the _OSG_SQUID_LOCATION_ environment
-  variable set
+To run your jobs on a list of specific execution sites, or avoid a set of 
+sites, use the `+DESIRED_Sites`/`+UNDESIRED_Sites` attributes in your job
+submit file. **These attributes should only be used as a last resort.** For
+example, it is much better to use feature attributes (see above) to make
+your job go to nodes matching what you really require, than to broadly
+allow/block whole sites. We encourage you to contact the facilitation team before taking this action, to make sure it is right for you. 
 
-- **HAS_TCSH** - Boolean specifying if the node has /bin/tcsh
+To avoid certain sites, first find the site names. You can find a 
+current list by querying the pool:
 
-- **HAS_XRDCP** - Boolean specifying if the node has _xrdcp_ in the default path
+    condor_status -af GLIDEIN_Site | sort -u
 
-- **HAS_TIMEOUT** - Boolean specifying if the node has _timeout_ in the default path
+In your submit file, add a comma separated list of sites like:
 
-- **HAS_R** - Boolean specifying if the node has R
+    +UNDESIRED_Sites = "ISI,SU-ITS"
 
-- **HAS_NUMPY** - Boolean specifying if the node has Python Numpy
+Those sites will now be exluded from the set of sites your job can
+run at.
```

Comment:
also say something about `+DesiredSites` or omit it above.

Reply:
Addressed in new commit